Sources/App/Migrations/027/UpdateRepositoryAddOwnerFields.swift

```diff
@@ -0,0 +1,20 @@
+import Fluent
+
+
+struct UpdateRepositoryAddOwnerNameAndOwnerAvatarUrlAndIsInOrganizationFlag: Migration {
```

Comment:
Nit: I'd keep file and struct name in sync, i.e. just call this `UpdateRepositoryAddOwnerFields` (I don't think it collides with any previous name). In hindsight we should probably have namespaces these with `Migration001` etc.

Reply:
> I'd keep file and struct name in sync 

Oops 😳

> (I don't think it collides with any previous name)

Presumable the unique index on `_fluent_migrations.name` should fail if names collide

> In hindsight we should probably have namespaced these with Migration001 etc.

Yeah, maybe [timestamps](https://github.com/hexpm/hexpm/tree/main/priv/repo/migrations ) instead of the incremental counter should work as well. Counters are more prone to conflicts.